packages/app/src/components/catalog/EntityPage.test.tsx

```diff
@@ -45,30 +48,30 @@ describe('EntityPage Test', () => {
 
   const mockedApi = {
     listWorkflowRuns: jest.fn().mockResolvedValue([]),
```

Comment:
It's nice to not have to list all methods out of the box, but it's a bit unfortunate that this mock function won't have proper types. It could be that we want to add some utility around that, but for now I think it's best for focus on moving over from `ApiRegistry` usage to a more testing-centered API without removing support for popular patterns.